pkg/controller/direct/cloudbuild/workerpool_controller.go

```diff
@@ -292,11 +291,7 @@ func (a *Adapter) Update(ctx context.Context, u *unstructured.Unstructured) erro
 		return fmt.Errorf("update workerpool status %w", mapCtx.Err())
 	}
 	// This value should not be updated. Just in case.
```

Comment:
Looks like we are no longer parsing the value, so maybe drop the comment (and possibly there's no point setting ExternalRef if we're just setting it back?)

Reply:
Sure